jobs/config.json

```diff
@@ -10097,21 +10097,14 @@
     ]
   },
   "pull-kubernetes-bazel-build": {
-    "args": [
-      "--build=//cmd/... //pkg/... //federation/... //plugin/... //third_party/... //examples/... //test/... //vendor/k8s.io/...",
```

Comment:
only move `--build` to prow for now since other args will still be the same?

Reply:
Hmm, we eventually want to move the args in anyhow though? There is only one other arg.
Jeff mentioned wanting a separate set of build/test targets per release in https://github.com/kubernetes/test-infra/pull/4940#discussion_r143591799, this is one way of solving that.

Reply:
(I mean `--build` and `--test`, but guess that doesn't matter that much)